decapCMS/config.yml

```diff
@@ -1,6 +1,6 @@
 backend:
   name: git-gateway
-  branch: master
+  branch: main
```

Comment:
You can specify whatever branch that you used. This is an example.

Reply:
Why "I used"?! It should coincide with the branch that git-gateway dev used, not me.

Reply:
I mean it's ok to choose any branch, you can refer to this guide: https://decapcms.org/docs/backends-overview/. The `master` is just a default branch I copied from a config template, without any further changes.